linux/LibSensors.c

```diff
@@ -208,6 +212,33 @@ void LibSensors_getCPUTemperatures(CPUData* cpus, unsigned int existingCPUs, uns
          if (r != 0)
             continue;
 
+         /* Map temperature values to Rockchip cores
+          *
+          *   littlecore -> cores 1..4
+          *   bigcore0   -> cores 5,6
+          *   bigcore1   -> cores 7,8
+          */
+         if (existingCPUs == 8 && String_eq(chip->prefix, "littlecore_thermal")) {
+            data[1] = temp;
+            data[2] = temp;
+            data[3] = temp;
+            data[4] = temp;
+            coreTempCount += 4;
+            continue;
+         }
+         if (existingCPUs == 8 && String_eq(chip->prefix, "bigcore0_thermal")) {
+            data[5] = temp;
+            data[6] = temp;
+            coreTempCount += 2;
+            continue;
+         }
+         if (existingCPUs == 8 && String_eq(chip->prefix, "bigcore1_thermal")) {
+            data[7] = temp;
+            data[8] = temp;
+            coreTempCount += 2;
+            continue;
+         }
```

Comment:
```suggestion
         if (existingCPUs == 8) {
            if (String_eq(chip->prefix, "littlecore_thermal")) {
               data[1] = temp;
               data[2] = temp;
               data[3] = temp;
               data[4] = temp;
               coreTempCount += 4;
               continue;
            }
            if (String_eq(chip->prefix, "bigcore0_thermal")) {
               data[5] = temp;
               data[6] = temp;
               coreTempCount += 2;
               continue;
            }
            if (String_eq(chip->prefix, "bigcore1_thermal")) {
               data[7] = temp;
               data[8] = temp;
               coreTempCount += 2;
               continue;
            }
         }
```

Reply:
Done.

Reply:
I fixed the remaining whitespace issue from your latest revision.